src/manage_nsfs/manage_nsfs_cli_utils.js

```diff
@@ -168,7 +191,7 @@ async function get_options_from_file(file_path) {
  * @param {object[]} access_keys
  */
 function has_access_keys(access_keys) {
-    return access_keys.length > 0;
+    return access_keys.length > 0 && access_keys[0].access_key;
```

Comment:
I'm not sure we want it, because the scenario of the access_keys object to be `{access_key: undefined, secret_key: undefined}` should not be a real data in a config, it is "by-product" of one of the functions.
In PR #8238 I'm suggesting to change it without changing this function.

Reply:
Either way is fine by me.